src/components/resource/edit-custom-title/resource-edit-custom-title.js

```diff
@@ -366,6 +371,19 @@ class ResourceEditCustomTitle extends Component {
                             )}
                           </div>
                           <CustomUrlFields />
+                          {accessStatusTypes?.request?.isPending
+                            ? (
+                              <Icon icon="spinner-ellipsis" />
+                            )
+                            : accessStatusTypes?.request?.records?.length > 0
+                              ? (
+                                <div data-test-eholdings-access-types-select>
+                                  <AccessTypeSelectField
+                                    accessStatusTypes={accessStatusTypes}
+                                  />
+                                </div>
+                              )
+                              : null}
```

Comment:
You can add just AND operand, but try to cover it in function `renderAccessTypeSelectField` to avoid nesting ternary operators

```suggestion
                            : accessStatusTypes?.request?.records?.length
                              && (
                                <div data-test-eholdings-access-types-select>
                                  <AccessTypeSelectField
                                    accessStatusTypes={accessStatusTypes}
                                  />
                                </div>
                              )}
```

Reply:
Changed to a ternary operator to return null value if condition is false.